app/src/main/java/org/mozilla/focus/activity/InfoActivity.java

```diff
@@ -0,0 +1,69 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.activity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import org.mozilla.focus.R;
+
+/**
+ * A generic activity that supports showing additional information in a WebView. This is useful
+ * for showing any web based content, including About/Help/Rights, and also SUMO pages.
+ */
+public class InfoActivity extends AppCompatActivity {
+    private static final String EXTRA_URL = "extra_url";
+    private static final String EXTRA_TITLE = "extra_title";
+
+    public static final Intent getIntentFor(final Context context, final String url, final String title) {
+        final Intent intent = new Intent(context, InfoActivity.class);
+
+        intent.putExtra(EXTRA_URL, url);
+        intent.putExtra(EXTRA_TITLE, title);
+
+        return intent;
+    }
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_info);
+
+        final String url = getIntent().getStringExtra(EXTRA_URL);
+        final String title = getIntent().getStringExtra(EXTRA_TITLE);
+
+        final WebView webView = (WebView) findViewById(R.id.webview);
+        // By default WebView will try to open links and redirects in an external browser (and sumo
+        // links usually use a redirect to get to the desired target page). Somehow, setting a default
+        // WebViewClient() is enough to disable that behaviour.
+        webView.setWebViewClient(new WebViewClient());
```

Comment:
I wonder whether we should go through WebSettings and there's nothing enabled by default that could lead to problems.

Do WebViews share a cache? E.g. is everything we browse to here available to the WebView in BrowserFragment? Should we clear the cache etc. in onDestroy()?